docs/guides/deployment/daemonize-worker.md

```diff
@@ -0,0 +1,137 @@
+---
+description: Learn how to set up a systemd service to daemonize a Prefect worker
```

Comment:
this guide appears to work as-is for a script calling `.serve` as well - I think that'd be worth testing and incorporating here assuming it works well

Reply:
Pushed changes to add .serve